apps/src/StudioApp.js

```diff
@@ -1372,18 +1372,11 @@ StudioApp.prototype.onReportComplete = function (response) {
   }
   this.lastShareUrl = response.level_source;
 
-  // Track GA events
   if (response.new_level_completed) {
-    trackEvent(
-      'Puzzle',
-      'Completed',
-      response.level_path,
-      response.level_attempts
-    );
-  }
-
-  if (response.share_failure) {
-    trackEvent('Share', 'Failure', response.share_failure.type);
+    trackEvent('puzzle', 'puzzle_completed', {
+      path: response.level_path,
+      value: response.level_attempts,
+    });
```

Comment:
There's a lot of magic strings hardwired throughout this PR. would it be worth the time to refactor them into external constants? It may be a diminishing return.

Reply:
Good point.  Might do this in a follow-up.